### PR TITLE
Introduced editorconfig as part of the generated code.

### PR DESCRIPTION
**Pull request by vraravam**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by vraravam** [comment]
This is one of the tasks identified in this [PR](https://github.com/jhipster/jhipster-dotnetcore/issues/332)

**Comment by nicolas63** [comment]
Looks good to me thanks for your contribution !

File: generators/common/templates/dotnetcore/README.md.ejs [hunk 1]
```diff
@@ -132,6 +132,13 @@ The `./src/<%= pascalizedBaseName %>/app/out` directory will contain your applic
 This will concatenate and minify the client CSS and JavaScript files. It will also modify `index.html` so it references these new files.
 <%_ } _%>
 
+## Code style / formatting
+
+To format the dotnet code, run
+
+    cd ./src/<%= pascalizedBaseName %>
```
Comment (nicolas63) [inline generators/common/templates/dotnetcore/README.md.ejs:139]: You can run dotnet format on root (on sln for format all solution)

**Review by nicolas63 — CHANGES_REQUESTED** [review]

File: generators/common/templates/dotnetcore/README.md.ejs (hunk 1 above)
Comment (vraravam) [reply generators/common/templates/dotnetcore/README.md.ejs]: Thanks - fixed in the latest commit

**Review by nicolas63 — APPROVED** [review]

**Pull request merged**